frontend/app/src/types/history/events/index.ts

```diff
@@ -14,7 +14,7 @@ const LiquityStakingEventExtraData = z.object({
 // ETH Transactions
 export const EvmTransaction = z.object({
   txHash: z.string(),
-  timestamp: z.number(),
+  timestamp: z.number().transform(timestamp => timestamp / 1000),
```

Comment:
Do we use these timestamps for editing later?
This transformation is converting the timestamp to seconds right?

Reply:
yes true, this way I can make the input consistent.
Ideally I think the backend should send everything in milliseconds.